webknossos/webknossos/dataset/dataset.py

```diff
@@ -1248,9 +1248,11 @@ def add_layer_from_images(
                     .offset(topleft)
                 )
             if pims_images.expected_shape != actual_size:
-                warnings.warn(
+                logger.info(
```

Comment:
why did you switch from warnings.warn to logger.info here?

Reply:
I think I experimented a bit and forget to change it back. It is now `warning.warn` again.